scripts/glean.py

```diff
@@ -188,11 +189,16 @@ def get_dependencies(self):
 
     def get_metrics(self) -> List[GleanMetric]:
         data = _cache.get_json(GleanApp.METRICS_URL_TEMPLATE.format(self.app["v1_name"]))
-        metrics = list(data.items())
-
+        metrics = [(m[0], {**m[1], "namespace": self.app["app_name"]}) for m in data.items()]
```

Comment:
```suggestion
        metrics = [(key, {**metricdict, "namespace": self.app["app_name"]}) for key, metricdict in data.items()]
```

I think this would be a bit clearer if we destructure in the `for` clause.